order-status/README.md

```diff
@@ -0,0 +1,63 @@
+# Overview
+
+This document explains the structure of a STAT **Order Status** request. Operation returns a list/array of statuses that have affected an order.  The most recent status code is the current status of the order.
+
+
+Endpoint: GET /order/{id}/status
+
+## Order Status Request
+
+List operation only in specced API.
+
+## Order Status Response
+
+### Order Status Response
+
+Response has two fields:
+| Field Name | Type | Description |
+| ---------- | ---- | ----------- |
+| current | OrderStatus | The current/official status object |
+| history | [OrderStatus] | History of statuses |
+
+### Order Status Object
+
+| Field Name | Type | Description |
+| ---------- | ---- | ----------- |
+| timestamp  | datetime | Timestamp for the order status (required) |
+| status_code | string | Enumerated status code (required) |
+| reason_code | string | Enumerated reason code for why the status was set (optional) |
+| reason_text | string | Textual description for why the status was set (optional) |
+| links | [Link] | list of references to documents, such as delivered asset, processing log, delivery manifest, etc. (required, may be empty) |
+
+### Enumerated status codes
+
+Providers must support these statuses.
+
+* received (indicates order received by provider and it passed format validation.)
+* accepted (indicates order has been accepted)
+* rejected (indicates order will not be fulfilled)
+* completed (indicates order had delivered imagery, must have been accepted first)
+* canceled (indicates provider was unable to collect/deliver tasked imagery)
+
+State machine intent:
+Received -> accepted or rejected.
+Accepted -> completed or cancel.
```

Comment:
The lifetime of the tasking order could be extended (especially for complex orders). Can one status be introduced to indicate that some actions are already occurring? E.g. Accepted -> In progress -> Completed.

Reply:
The intention is that extension statuses can be inserted into the flow, but they are provider specific.  An in_progress status would be either an optional or extension-provided status.